src/containerapp/azext_containerapp/_models.py

```diff
@@ -21,7 +21,8 @@
         "appLogsConfiguration": None,
         "customDomainConfiguration": None,  # CustomDomainConfiguration,
         "workloadProfiles": None,
-        "InfrastructureResourceGroup": None
+        "InfrastructureResourceGroup": None,
```

Comment:
I -> i?

Reply:
When change the I->i here, also needs to fix this line:
https://github.com/Azure/azure-cli-extensions/blob/607bb0c220d002f06319c7bb602683ca12d7b5d2/src/containerapp/azext_containerapp/containerapp_env_decorator.py#L48